XCUITests/HistoryTests.swift

```diff
@@ -5,11 +5,15 @@
 import XCTest
 
 let webpage = ["url": "www.mozilla.org", "label": "Internet for people, not profit — Mozilla", "value": "mozilla.org"]
+let allWebpages = ["url1": "www.mozilla.org", "label1": "Internet for people, not profit — Mozilla", "value1": "mozilla.org",
+                "url2": "www.twitter.org", "label2": "Twitter", "value2": "twitter.com",
+                "url3": "www.youtube.com", "label3": "Home - YouTube", "value3": "youtube.com"]
+let allHistoryEntrys = ["label1", "label2", "label3"]
```

Comment:
I think we can do this in a simplier way but need to think about it...so far it is kind of confusing...could you try a different approach?

Reply:
Let's see if you agree with this solution...Create only one array:
`let oldHistoryEntries: [String] = ["Internet for people, not profit — Mozilla", "Twitter", "Home - YouTube"]`

And go through it like this:
```
for entry in oldHistoryEntries {
XCTAssertTrue(app.tables.cells.staticTexts[entry].exists)
}
```

I think that may simplify the test since you were using allWebpages string only to create after it the allHistoryEntries, right? 
Please correct me if I'm wrong or if you think there is a better way to do it...
Thanks!